appengine/flexible/tasks/README.md

```diff
@@ -61,47 +62,44 @@ Verify the index page is serving:
     gcloud app browse
 
 The App Engine app serves as a target for the push requests. It has an
-endpoint `/set_payload` that that stores the payload from the HTTP POST data in
-Cloud Datastore. The payload can be accessed in your browser at the
- `/get_payload` endpoint with a GET request.
+endpoint `/log_payload` that reads the payload (i.e., the request body) of the
+HTTP POST request and logs it. The log output can be viewed with:
+
+    gcloud app logs read
 
 ## Running the Samples
 
 The project ID must be specified either as a command line argument using
 `--project-id`, or by editing `DEFAULT_PROJECT_ID` within `task_snippets.py`.
 
-Set the environment variables:
+Set environment variables:
+
+First, your project ID:
 
     export PROJECT_ID=my-project-id
-    export LOCATION_ID=us-central1
-    export QUEUE_ID=my-appengine-queue # From queue.yaml
 
-View all queues:
+Then the queue ID, as specified at queue creation time. Queue IDs already
+created can be listed with `gcloud alpha tasks queue list`.
 
-     python app_engine_queue_snippets.py --api_key=$API_KEY list-queues --project_id=$PROJECT_ID --location_id=$LOCATION_ID
+    export QUEUE_ID=my-appengine-queue
 
-Set the queue name as an environment variable:
+And finally the location ID, which can be discovered with
+`gcloud alpha tasks queue describe $QUEUE_ID`, with the location embedded in the
+"name" value (for instance, if the name is
+"projects/my-project/locations/us-central1/queues/my-appengine-queue", then the
+location is "us-central1").
 
-    export QUEUE_NAME=projects/$PROJECT_ID/locations/$LOCATION_ID/queues/$QUEUE_ID
+    export LOCATION_ID=us-central1
 
 Create a task, targeted at the `set_payload` endpoint with a payload specified:
 
-   python app_engine_queue_snippets.py --api_key=$API_KEY create-task --queue_name=$QUEUE_NAME --payload=hello
-
-Now view that the payload was received and verify the count and payload:
-
-    http://your-app-id.appspot.com/get_payload
-
-Create a task that will be scheduled for a few seconds in the future using
-the `--in_seconds` flag:
-
-    python app_engine_queue_snippets.py --api_key=$API_KEY create-task --queue_name=$QUEUE_NAME --payload=hello --in_seconds=30
+   python create_app_engine_queue_task.py --project=$PROJECT_ID --queue=QUEUE_ID --location=LOCATION_ID --payload=hello
```

Comment:
curious why the queue_name parameter was removed in favor of breaking it out into three components? While I think the naming conventions of "queue name" vs. "queue" vs. "queue ID" are hard to follow, they're apparently in line with OnePlatform. Further, when we want to add more params like headers, App Engine routing, relative URL, and response view, the command will get pretty long.

Reply:
I removed queue_name because gcloud queues list doesn't show the queue name. There's no real way to discover it without hitting the API separately. The queue name construct is still visible in the code itself, which seems appropriate for a sample.

Reply:
But the `describe` command does, which is what you're using to get the location anyway.

Reply:
Oh, that's right. I think using the "queue_name" is an awkward abstraction, but since "describe" shows the queue_name I could add it back in. I'll reconsider.

Reply:
I discussed with Jon and he told me that our other samples that have resource paths under the hoods use a similar "pass in the components on the command line, and assemble the full path in the code itself" system. So, I think I'll keep it like this for now. I think it's awkward to have the user use "describe" to look at the resource path just to get the location out of it, but I've been assured that the resource path will be included in "list" output soon at which point that will no longer be necessary.